scripts/deployment/upload_model_to_huggingface.py

```
@@ -0,0 +1,1220 @@
+Upload your custom-trained emotion detection model to HuggingFace Hub 
+so it can be used in production deployment.
+"""
+
+import os
+import sys
+import json
+import shutil
+from typing import Optional
+
+# Use built-in generics for Python 3.9+ (PEP 585)
+if sys.version_info >= (3, 9):
```

Ruff (0.12.2)

Version block is outdated for minimum Python version

Remove outdated version block

(UP036)